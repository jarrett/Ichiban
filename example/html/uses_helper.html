<p>This page uses a helper. 2 x 2 is <%= multiply 2 %>.</p>