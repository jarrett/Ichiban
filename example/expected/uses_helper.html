<!DOCTYPE html>
<html>
  <head>
    <title>Example Site</title>
  </head>
  
  <body>
    <h1>Example Site</h1>
    
    <p>This page uses a helper. 2 x 2 is 4.</p>
  </body>
</html>